src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,29 @@
+package duke.task;
+
+public class Deadline extends Task {
+
+    public static final String DEADLINE_LABEL = "D";
```

Comment:
If this variable doesn't need to be accessed outside of this class, it is probably better not to declare it as public